src/app.js

```diff
@@ -71,8 +72,9 @@ try {
 }
 
 // routing
-app.use('/', indexRouter)
 app.use(express.static(staticPath))
+app.use('/', indexRouter)
```

Comment:
Good call ordering this after static files.  Should we also move `app.use(express.json())` after static?  Or should that middleware be called only specifically when needed per route?

Reply:
I think that's a good idea actually